scripts/build-bundle

```diff
@@ -140,9 +145,9 @@ check_addon_chart_version_matching() {
   ls -alht ${CHARTS_DIR}
 
   echo "addon template files"
-  ls -alht ${addons_path}/pkg/templates
+  ls -alht ${addons_path}
```

Comment:
Should we change this to `ls -alht ${addons_path}/*.yaml` ? I think we just care the *.yaml, right?

Reply:
this is to only to help with debugging.. the subsequent loop limits itself to `*.yaml`